File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int king=1,queen=1,look=2,bee=2,night=2,phone=8;
		
		int iking,iqueen,ilook,ibee,inight,iphone;
		
		iking = input.nextInt();
		iqueen = input.nextInt();
		ilook = input.nextInt();
		ibee = input.nextInt();
		inight = input.nextInt();
		iphone = input.nextInt();
		
		System.out.print(king-iking+" ");
		System.out.print(queen-iqueen+" ");
		System.out.print(look-ilook+" ");
		System.out.print(bee-ibee+" ");
		System.out.print(night-inight+" ");
		System.out.println(phone-iphone);
		
	}
}

File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.cc
#include<stdio.h>

int main() {

	int king=1, queen=1, look=2, bishop=2, night=2, phone=8;
	int n1, n2, n3, n4, n5, n6;
	scanf("%d %d %d %d %d %d", &n1, &n2, &n3, &n4, &n5, &n6);

	printf("%d ", king - n1);
	printf("%d ", queen - n2);
	printf("%d ", look - n3);
	printf("%d ", bishop - n4);
	printf("%d ", night - n5);
	printf("%d", phone - n6);

	

	return 0;
}